xmls/xml_include/xml_building_blocks/platforms.xml

```diff
@@ -30,6 +30,37 @@
       $(append_to_setup_csh)
     ]]></csh>
   </platform>
+  <platform name="ncrc6.intel23">
+    <freVersion>$(FRE_VERSION)</freVersion>
+    <compiler type="intel-classic" version="2023.2.0"/>
+    <project>$(NCRC_GROUP)</project>
+    <directory stem="$(FRE_STEM)">
+      <root>/gpfs/f6/$(project)/scratch/$USER/$(stem)</root>
+      <scripts>$(rootDir)/$(name)/$(platform)-$(target)/scripts</scripts>
+      <state>$(rootDir)/$(name)/$(platform)-$(target)/state</state>
+      <src>$(rootDir)/$(name)/src</src>
+      <exec>$(rootDir)/$(name)/$(platform)-$(target)/exec</exec>
+      <archive>$(rootDir)/$(name)/$(platform)-$(target)/archive</archive>
+      <stdout>$(rootDir)/$(name)/$(platform)-$(target)/stdout</stdout>
+      <work>/gpfs/f6/$(project)/scratch/$USER/work/$FRE_JOBID</work>
+    </directory>
+    <property name="EXEC_ROOT"         value="/gpfs/f6/$(project)/scratch/$USER/$(stem)/$(BUILD_DATE)"/>
+    <property name="F2003_FLAGS"       value=" -DINTERNAL_FILE_NML -g "/>
+    <property name="AWG_INPUT_HOME"    value="$(AWG_INPUT_HOME_GAEA)"/>
+    <property name="NB_ROOT"           value="/nbhome/$USER/$(FRE_STEM)$(PLATINFO)/$(name)"/>
+    <property name="ARCHIVE_NNZ"        value=""/>
+    <property name="REFERENCE"         value=""/>
+    <property name="FMS_ARCHIVE_ROOT"    value=""/>
+    <property name="CMIP6_ARCHIVE_ROOT"  value=""/>
+    <property name="NCORE_PER_NODE"      value="128"/>
```

Comment:
I don't know if this even gets used anywhere, but technically, is c6 192 cores per node?

Reply:
@andrew-c-ross, that’s correct. I thought I had changed it, but apparently, I missed it. Good catch.